ServiceBus/SharedLibs/DataContracts/Product.cs

```diff
@@ -6,6 +6,7 @@ namespace SharedLibs.DataContracts
     /// <summary>
     /// Datacontract for product
     /// </summary>
+    //TODO Is obsolote datacontract or not? 

```

Comment:
We should provide the customer with all possible information. Only internal (our) data should not be shared. Right now we have no internal information stored in products, therefore we can add everything.